pom.xml

```diff
@@ -244,7 +244,7 @@
         <jts.spatial4j.version>1.15.0</jts.spatial4j.version>
         <jts.version>1.14.0</jts.version>
         <jvnet-ogc.version>2.6.1</jvnet-ogc.version>
-        <karaf.version>4.2.2</karaf.version>
+        <karaf.version>4.2.3</karaf.version>
```

Comment:
✏️ You can remove these lines:
https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/common-bin.xml#L28-L29
and this file
https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/etc/jre.properties
since the workaround was for a bug fixed in Karaf 4.2.3.